gui/widgets/qopenglmousetracker_t.cpp

```diff
@@ -0,0 +1,105 @@
+//
+// Created by acaramizaru on 7/25/23.
+//
+
+#include "catch_wrapper.hpp"
+#include "qt_test_utils.hpp"
+
+#include "rendering/rendering.hpp"
+
+using namespace sme::test;
+
+#include "qopenglmousetracker.h"
+
+#include "sme/logger.hpp"
+
+#if defined(Q_OS_WIN) || defined(Q_OS_UNIX)
+static const char *tags{"[gui/widgets/QOpenGLMouseTracker][gui][opengl]"};
+#else
+static const char *tags{"[gui/widgets/QOpenGLMouseTracker][gui][~opengl]"};
+#endif
```

Comment:
the tags of the tests in the compiled binary shouldn't depend on the OS this is compiling on, instead the choice of which tags to run should be done in the ci scripts at runtime (e.g. here https://github.com/spatial-model-editor/spatial-model-editor/blob/f6c24e6a9e41b840bd908d25d52ed1ef990559e0/ci/macos-gui.sh#L32) (i.e. the tags in the code here are labels which can then be used to select tests at run-time)

Reply:
So you still want the test to be available in C.I. for Mac even if the functionality is not available there? OK.

Reply:
To clarify how the tags work:
- the tag you put in the source code is a label that you can use to select (or exclude) this tag when running the tests
- when you then run the tests you can choose which tags to include or exclude, e.g.
  - `./tests [opengl]` run only the tests with an [opengl] tag
  -  `./tests ~[opengl]` run all tests excluding those with an [opengl] tag
  - more info here: https://github.com/spatial-model-editor/spatial-model-editor/blob/main/test/README.md